src/redux/Mission/missionSlice.js

```diff
@@ -0,0 +1,63 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+export const fetchmission = createAsyncThunk('missions/fetchmission', async () => {
+  const response = await fetch('https://api.spacexdata.com/v3/missions');
+  const { data } = response.json();
+  return data;
```

Comment:
The missions part of your application is broken and is not loading any missions. Kindly fix this issue. The bug might have been introduced in the latest changes you made in your application. From what I tested, you are trying to destructure the data from the response but the data is already available as an array. ⬇️
![image](https://user-images.githubusercontent.com/33268927/224474128-8deed730-c06a-405d-a3df-4722117cf90f.png)